google-cloud-spanner/src/main/java/com/google/cloud/spanner/AbstractStructReader.java

```diff
@@ -43,6 +43,8 @@ public abstract class AbstractStructReader implements StructReader {
 
   protected abstract long getLongInternal(int columnIndex);
 
+  protected abstract float getFloatInternal(int columnIndex);
```

Comment:
These methods should also have a default implementation to prevent this from becoming a breaking change.

Reply:
Thanks, made the change